spec/models/suggested_councillor_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper'
+
+RSpec.describe SuggestedCouncillor, type: :model do
+  it "should require a name" do
```

Comment:
You might be able to make these tighter using `subject` (see http://www.betterspecs.org/#subject ) and `be_valid` (using [Rails method `.valid?`](http://api.rubyonrails.org/classes/ActiveRecord/Validations.html#method-i-valid-3F) with [rspec predicate matchers](https://relishapp.com/rspec/rspec-expectations/v/3-6/docs/built-in-matchers/predicate-matchers))

Do you think something like this would be a bit clearer + more succinct?

```
describe "is invalid without an email" do
  subject { SuggestedCouncillor.new(name: "Jane", email: nil, authority_id: 1) }
  it { is_expected.to_not be_valid }
end

describe "is invalid without a name" do
  subject { SuggestedCouncillor.new(name: nil, email: "example@email.org", authority_id: 1) }
  it { is_expected.to_not be_valid }
end

describe "is invalid with a blank name" do
  subject { SuggestedCouncillor.new(name: "", email: "example@email.org", authority_id: 1) }
  it { is_expected.to_not be_valid }
end
````

Or maybe it's time to start using a library for these common validation tests we have https://github.com/thoughtbot/shoulda-matchers#activemodel-matchers